Xamarin.Essentials/Types/WindowSize.shared.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Xamarin.Essentials
+{
+    public readonly struct WindowSize : IEquatable<WindowSize>
+    {
+        public WindowSize(double width, double height)
+        {
+            Width = width;
+            Height = height;
+        }
+
+        public double Width { get; }
+
+        public double Height { get; }
+
+        public bool Equals(WindowSize other) =>
+            Width.Equals(other.Width) && Height.Equals(other.Height);
+
+        public override bool Equals(object obj) =>
+            (obj is WindowSize windowSize) && Equals(windowSize);
+
+        public override int GetHashCode() =>
+            (Width, Height).GetHashCode();
+
+        public override string ToString() =>
+            $"{nameof(Width)} : {Width} \n {nameof(Height)}:{Height}";
```

Comment:
`{Environment.NewLine}` instead of `\n`?

Reply:
Do we need a new size type? Can we not just use `Size`?

Reply:
If anything, I would copy corefx: https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Primitives/src/System/Drawing/Size.cs#L192

Reply:
@mattleibow I tried to use this `Size`, that you mentioned above. But I got compile errors from `netStandard1.0`